cmd/executor/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/http"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	ehandler "github.com/kubeflow/kfserving/pkg/executor"
+	perrors "github.com/pkg/errors"
+	"golang.org/x/net/http2"
+	"golang.org/x/net/http2/h2c"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var (
+	predictor   = flag.String("predictor", "", "The FQDN of the predictor service")
```

Comment:
nit: can these be 
```
predictor_url
preprocessor_url
postprocessor_url
```

Reply:
They are not presently URLs but hostnames. They are used like:
```
target := &url.URL{
			Scheme: "http",
			Host:   eh.postprocess,
		}
```

predictor_host
preprocessor_host
postprocessor_host
?

Reply:
Makes sense.

Perhaps 
```
predictor
preprocessor
postprocessor
```
is good enough :)